res/controllers/Pioneer-DDJ-400-script.js

```diff
@@ -0,0 +1,684 @@
+// Pioneer-DDJ-400-script.js
+// ****************************************************************************
+// * Mixxx mapping script file for the Pioneer DDJ-400.
+// * Authors: Warker, nschloe, dj3730, jusko
+// * Reviewers: Be-ing, Holzhaus
+// * Manual: https://manual.mixxx.org/2.3/en/hardware/controllers/pioneer_ddj_400.html
```

Comment:
Should we remove the `/en/` part of the URL? I guess a redirect (301) will add this implicitly.